src/elm/Session/LoggedIn.elm

```diff
@@ -78,6 +83,8 @@ init shared accountName flags =
     , Cmd.batch
         [ Api.Graphql.query shared (Profile.query accountName) CompletedLoadProfile
         , Api.Graphql.query shared (Community.settingsQuery flags.selectedCommunity) CompletedLoadSettings
+        , Ports.getRecentSearches () -- run on the page refresh, duplicated in `initLogin`
```

Comment:
I have this duplicated because there are two functions to initialize the `LoggedIn` model: `init` and `initLogin`. One works for signing-in with the passphrase and the other works for the page refresh.